packages/tracing-processing/src/services/processingService.ts

```diff
@@ -0,0 +1,57 @@
+import { genericInternalError } from "pagopa-interop-tracing-models";
+import { BucketService } from "./bucketService.js";
+import { DBService } from "./db/dbService.js";
+import { ProducerService } from "./producerService.js";
+
+export const processingServiceBuilder = (
+  dbService: DBService,
+  bucketService: BucketService,
+  producerService: ProducerService,
+) => {
+  const enrichFile = (records: Array<unknown>, purposes: Array<unknown>) => {
```

Comment:
can you remove this implementation for now?
it's a bit too detailed for a scaffold and I may oversee some changes in the implementation PR